frame/lending/src/lib.rs

```diff
@@ -1009,17 +998,25 @@ pub mod pallet {
 			}
 
 			let borrow_asset = T::Vault::asset_id(&market.borrow)?;
-			let borrow_limit_value = Self::get_borrow_limit(market_id, debt_owner)?;
+
+			let borrow_limit = Self::get_borrow_limit(market_id, debt_owner)?;
 			let borrow_amount_value = Self::get_price(borrow_asset, amount_to_borrow)?;
+			dbg!("{:?}", borrow_limit);
```

Comment:
only printing values, may not be easy to read, should we add names too?

Reply:
oh. forget about.